clients/js/main.ts

```diff
@@ -656,6 +666,10 @@ function parseAddress(chain: ChainName, address: string): string {
     throw Error("SUI is not supported yet");
   } else if (chain === "aptos") {
     throw Error("APTOS is not supported yet");
+  } else if (chain === "pythnet") {
+    throw Error("APTOS is not supported yet");
```

Comment:
PYTHNET?

Reply:
This was handled by the rebase to dev.v2